frontend.yaml

```diff
@@ -0,0 +1,37 @@
+#The command to run this is 'kubectl apply -f frontend.yaml'
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frontend-deployment
+  namespace: kube1
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: frontend
+  template:
+    metadata:
+      labels:
+        app: frontend
+    spec:
+      containers:
+        - name: frontend
+          image: ghcr.io/expruc/kubernetes-gradutaion-drill/k8s-final-drill:frontend
+          ports:
+            - containerPort: 8080
+          env:
+            - name: PORT
+              value: "8080" 
+            - name: BACKENDURL
+              value: "http://backend-service:8082"
+          resources:
+            limits:
+              cpu: "50m"
+              memory: "64Mi"
+          readinessProbe:
+            httpGet:
+              path: /check-readiness
+              port: 8080
+            #Honestly I didn't know how long the delay should be so I guessed
```

Comment:
usually we will test how much it takes for the pod to start (through logs for example) and will know the delay